compiler/src/codegen/compcore.re

```diff
@@ -2465,6 +2465,102 @@ let compile_primn = (wasm_mod, env: codegen_env, p, args): Expression.t => {
         Expression.const(wasm_mod, const_void()),
       ],
     )
+  | WasmMemoryCompare =>
```

Comment:
This is an implementation of `memcmp`: https://github.com/gcc-mirror/gcc/blob/master/libiberty/memcmp.c (public domain).

Reply:
inb4 why is this in the compiler and not just Grain code: Ideally this should always be inlined, and we can't yet guarantee that it gets inlined across files, but it will always be inlined as a compiler primitive.